lib/podlet.js

```diff
@@ -818,6 +823,55 @@ export default class PodiumPodlet {
         return this.#view(incoming, data, ...args);
     }
 
+    /**
+     * I'm imaginging this as a helper method to be used in conjunction with render/podiumSend.
+     *
+     * Eg.
+     * app.get("/", (req, res) => {
+     *	res.podiumSend(res.podiumIsolate("<div>content to render</div>"));
```

Comment:
```suggestion
     *	res.podiumSend("<div>content to render</div>", this.isolate);
```
If we in Express add a parameter to `.podiumSend` which is a wrapper / isolate function I think that can make for a nice way of doing this.

It also makes it possible to implement something similar for Fastify where you can add a config option to the route(s) that is the wrapper function and make it happen in a `onSend` hook.
```js 

fastify.addHook('onSend', (request, reply, payload) => {
 if (reply.context.config.podlet.isolate) {
  return reply.context.config.podlet.isolate(payload);
 }
})
fastify.get("/", { config: { podlet: { wrapper: this.isolate } } }, async (req, reply) => {
 reply.podiumSend("<div>...</div>");
 return reply;
}
```

Reply:
Right now the arguments to `podiumSend` get passed on to the [view render function](https://github.com/podium-lib/podlet/blob/994ee59567ebf15ab1410b2a40bfba3fcf50d4ed/lib/podlet.js#L911-L912). Changing that would be a breaking change and force us to redesign how we pass arguments to the render function.

Reply:
Right, I wasn't aware of that. The docs on `.render` argument `args` is a bit vague "additional args depending on the template and what values it accepts".
🤔 the `.render` method isn't documented either.

Reply:
Yes, as I see it, we can't really touch podiumSend without a breaking change and it would be nice to avoid.

The problem with methods on the podlet, eg. podlet.render is that you will always need to pass httpIncoming. By adding an additional method on the res/reply objects, we can inject httpIncoming and nicify up the api a bit.